src/features/WordTable/index.tsx

```diff
@@ -80,13 +96,15 @@ const WordTable = ({
         ):
           <div className = {css.emptyState}><Text type="subheading" > Sorry, it seems you don't have any words </Text></div>}
       </div>
-      {activeCard && (
+
+      {
+        activeCards.length && (
         <WordPanel
-          cardInfo={activeCard}
-          saveCard={() => moveCard(activeCard.id)}
-          deleteCard={() => deleteCard(activeCard.id)}
-        />
-      )}
+          cardInfo={activeCards[activeCards.length-1]}
+          saveCard={() => cardLoop(activeCards, moveCard)}
+          deleteCard={() => cardLoop(activeCards, deleteCard)}
```

Comment:
You don't need to pass in activeCards as a param here - it's already a global state

Reply:
nvm I forgot what this was part of